### PR TITLE
python 3 compatibility

### PR DESCRIPTION
**Pull request by dirk-thomas**

### CODE REVIEW

**Comment by tfoote** [comment]
+1

File: src/genmsg/gentools.py [hunk 1]
```diff
@@ -87,8 +87,7 @@ def compute_md5_text(msg_context, spec):
             sub_md5 = compute_md5(msg_context, sub_spec)
             buff.write("%s %s\n"%(sub_md5, name))
     
-    value = buff.getvalue().strip() # remove trailing new line
-    return value.encode()
+    return buff.getvalue().strip() # remove trailing new line
```
Comment (dirk-thomas) [inline src/genmsg/gentools.py:90]: This reverts on change of #35 which turned out to be the wrong way to address the problem.

**Comment by dirk-thomas** [comment]
@tfoote @wjwwood Please review.

**Comment by tfoote** [comment]
+1

**Comment by wjwwood** [comment]
+1

**Pull request merged**